requirements/specifications/media/media-info.md

```diff
@@ -9,30 +9,27 @@ See [Firebolt Requirements Governance](../../governance.md) for more info.
 | Anthony Borzota | Comcast        |
 | Jeremy LaCivita | Comcast        |
 | Stuart Harris   | Sky            |
-| Farhan Mood     | Liberty Global            |
+| Farhan Mood     | Liberty Global |
 
 ## 1. Overview
-App developers need to know which audio and video formats can be successfully played on a Firebolt device.
 
-Video formats include... 
+App developers need to know which audio and video formats can be successfully played on a Firebolt device and its attached display.
 
-**TODO**: Rewrite this section.
+An audio format includes values such as the audio container, codec, codec level, and bit rate of the audio stream.
 
-Apps may need to know what media format, e.g. HDR profile or video codec, is currently playing.
+A video format may include values such as the video container, codec, HDR profile, and resolution of the video stream.
 
-An app may need to check IP video playback that it has initiated to see if Dolby Audio is in fact active (as opposed to just present in the encoded data) in order to display metadata or badges to the user. Apps may also be playing [user-cultivated media](./media-access.md), and therefor have no metadata about the format of the files.
+Further, apps may also need to know about the media format currently playing in the media pipeline.
+
+An app may need to check IP video playback that it has initiated to see if Dolby Audio is in fact active (as opposed to just present in the encoded data) in order to display metadata or badges to the user. Apps may also be playing [user-cultivated media](./media-access.md), and therefore have no metadata about the format of the files.
 
 A first-party app may need to check all [Media Pipelines](./media-pipeline.md) for media format characteristics to display a global badge.
 
 Additionally, apps may need to know what is supported by the device, *before* initiating any media playback.
 
-As a W3C-based platform, there is no standard API for detecting media
-formats or codecs that works in all cases, short of inspecting the bytes
-of the media in the app itself.
+As a W3C-based platform, there is no standard API for detecting media formats or codecs that works in all cases, short of inspecting the bytes of the media in the app itself.
```

Comment:
This is actually not true anymore, I was wrong. There's an API called MediaCapabilities that does a bunch of this

We should probably remove this paragraph from the overview.

I'm tempted to add a section to the spec that says browsers on Firebolt platforms MUST use these APIs to fulfill the W3C MediaCapabilities API.